### ISSUE TITLE
For specimen, only one image showing in viewer when there are multiple images

### ISSUE THREAD

**Issue by doricon** [paleobot/pbot-dev#183]
I added two images to a specimen: http://128.196.196.17/query/specimen/c44cbcac-e6e6-4623-977e-4a0131ebb603?includeImages=true&includeDescriptions=true&includeOTUs=true

But only one of the images is showing on the specimen page. 

Also, the image captions don't show with the images. 

(I labeled this as bug because I thought this worked before, but maybe I am remembering wrong and it just hasn't been implemented yet)

**Comment by doricon** [comment]
Update: OTUs don't show images, so yes, the image carousel is not working for me

**Comment by doricon** [comment]
Oops, you're right. Carousel is working, I failed to find the hover to change image. 

The OTU query output is a post-hackathon thing since it hasn't been implemented yet (I couldn't remember where the carousel currently was), but eventually we will want the specimen images (and some other specimen and collection summary data) to show on the OTU page. Back in the Tuscon meeting we worked out how the image carousel would prioritize images from different specimens, which is why we have the Category field on the image form. Ooo now that we understand Git, I could put in "enhancement" issues for these things so they don't get lost to time! :)
